jupyter-notebooks/api_guides/features_api/planet_sdk_features_demo.ipynb

```diff
@@ -0,0 +1,742 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "33202e83",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lake_poly = [[[31.23905453, 29.99579775], [31.26630889, 29.99579775], [31.26630889, 30.01908084], [31.23905453, 30.01908084], [31.23905453, 29.99579775]]]\n",
+    "institute_poly = [[[31.23344423, 30.05109414], [31.25491533, 30.05109414], [31.25491533, 30.06461941], [31.23344423, 30.06461941], [31.23344423, 30.05109414]]]\n",
+    "\n",
+    "cairo_fc = {\n",
+    "    \"type\": \"FeatureCollection\",\n",
+    "    \"features\": [\n",
+    "        {\n",
+    "            \"id\": \"0\",\n",
+    "            \"type\": \"Feature\",\n",
+    "            \"properties\": {\n",
+    "                \"title\": \"lake_area\",\n",
+    "                \"name_str\": \"area_1\",\n",
+    "                \"extra_property_1\": 'extra property 1 here',\n",
+    "                \"extra_property_2\": 'extra property 2 here'},\n",
+    "            \"geometry\": {\"coordinates\": lake_poly, \"type\": \"Polygon\"},\n",
+    "        },\n",
+    "        {\n",
+    "            \"id\": \"1\",\n",
+    "            \"type\": \"Feature\",\n",
+    "            \"properties\": {\n",
+    "                \"title\": \"institute_area\",\n",
+    "                \"name_str\": \"area_2\",\n",
+    "                \"extra_property_1\": 'extra property 1 here',\n",
+    "                \"extra_property_2\": 'extra property 2 here'},\n",
+    "            \"geometry\": {\"coordinates\": institute_poly, \"type\": \"Polygon\"},\n",
+    "        },\n",
+    "]}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f4f8ce61",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read your geojson path, then send it to a geo_dict.\n",
+    "geojson_file = 'PATH TO YOUR GEOJSON FILE'\n",
+    "\n",
+    "gdf = gpd.read_file(geojson_file)\n",
+    "\n",
+    "local_fc = gdf.to_geo_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "809f257d",
+   "metadata": {},
+   "source": [
+    "___"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70faedbb",
+   "metadata": {},
+   "source": [
+    "### Name and create your FeatureCollection in the Features API\n",
+    "\n",
+    "Before you upload your FeatureCollection items, you need to create the Collection with the Features API. Set your `collection_title`, which will serve as the title or name of the whole FeatureCollection once its uploaded. You may optionally set a description for your Collection. Then we can create our new Collection using `pl.features.create_collection`, which returns the newly created Collections's `collection_id`. The `collection_id` is needed to add items to the Collection."
```

Comment:
It's probably worth mentioning the alternative option: If you have an existing collection, you can add your features to that.